suite-common/message-system/src/utils.ts

```diff
@@ -1,10 +1,4 @@
 import * as semver from 'semver';
```

Comment:
I would suggest to keep name of file to `messageSystemUtils.ts` or rename test file to just `utils.test.ts` to keep names same.

Reply:
69a7ed894851836f3c5562d08e1396f484391e9f